### PR TITLE
Feat: 1421 add methane capture project type

### PR DESCRIPTION
**Pull request by JoshLarouche**
[zenhub 1421](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1421)

### CODE REVIEW

File: schema/test/unit/tables/project_type_test.sql [hunk 1]
```diff
@@ -72,8 +72,8 @@ select results_eq(
   $$
     select count(*) from cif.project_type
   $$,
-  -- 11 project types + project type added in earlier test = 12
-  ARRAY['12'::bigint],
+  -- 12 project types + project type added in earlier test = 12
```
Comment (BCerki) [inline schema/test/unit/tables/project_type_test.sql:75]:
```suggestion
  -- 12 project types + project type added in earlier test = 13
```

File: schema/sqitch.plan [hunk 2]
```diff
@@ -299,3 +299,4 @@ computed_columns/form_change_as_project_attachment [tables/attachment tables/for
 mutations/update_milestone_form_change [mutations/update_milestone_form_change@1.7.0] 2023-05-10T23:41:13Z Sepehr Sobhani <sepehr.sobhani@gov.bc.ca> # Remove total eligible expenses from the form data when milestone type is not General
 mutations/update_form_change [mutations/update_form_change@1.6.0] 2023-04-19T23:31:07Z Gurjeet Matharu <gurjeet@matharuco.com> # fix in update_form_change to handle a null operation for archived forms.  Should now set to update.
 @1.8.0 2023-05-16T21:39:16Z Dylan Leard <dylan@button.is> # release v1.8.0
+tables/project_type_001 2023-05-11T21:32:24Z Josh Larouche <joshua.larouche@gov.bc.ca> # adding project type 001
```
Comment (BCerki) [inline schema/sqitch.plan:302]: This comment could be a little more descriptive, maybe something along the lines of this example one:

`tables/report_type_003 2023-01-31T00:01:02Z Thomas Mastromonaco <thomas@button.is> # Add summary report to the report type table`

File: app/components/Form/ProjectForm.tsx [hunk 3]
```diff
@@ -313,14 +313,20 @@ const ProjectForm: React.FC<Props> = (props) => {
...
+                type: "string",
+                title: node.name,
+                enum: [node.name],
+                value: node.name,
+              };
+            })
+            .sort((a, b) => {
+              if (b.value === "Other") {
+                return -1;
+              } else return 0;
```
Comment (BCerki) [inline app/components/Form/ProjectForm.tsx:328]: Cool, this looks great

**Review by BCerki — DISMISSED** [review]
🤸

**Review by Sepehr-Sobhani — APPROVED** [review]
🤩

**Pull request merged**